examples/jaeger/build.gradle

```diff
@@ -3,12 +3,33 @@ plugins {
 }
 
 description = 'OpenTelemetry Examples for Jaeger Exporter'
-ext.moduleName = "io.opentelemetry.examples.jaeger"
+ext {
+    moduleName = "io.opentelemetry.examples.jaeger"
+    testcontainersVersion = "1.14.2"
+}
+
+sourceCompatibility = 1.7
+targetCompatibility = 1.7
+
+compileTestJava {
+    sourceCompatibility = 1.8
+    targetCompatibility = 1.8
+}
 
 dependencies {
     compile("io.opentelemetry:opentelemetry-api:${opentelemetryVersion}")
     compile("io.opentelemetry:opentelemetry-sdk:${opentelemetryVersion}")
     compile("io.opentelemetry:opentelemetry-exporters-jaeger:${opentelemetryVersion}")
     compile("io.grpc:grpc-protobuf:1.28.0")
     compile("io.grpc:grpc-netty-shaded:1.28.0")
+
+    implementation platform("org.testcontainers:testcontainers-bom:${testcontainersVersion}")
```

Comment:
My understanding of "platform" and boms is weak. This doesn't make the jaeger exporter depend on testcontainers, does it? Should this be `testImplementation` ?

Reply:
That's a good catch - it should be ```testImplementation``` only.  I've changed it :)
I assume you meant jaeger example not jaeger exporter?
If we had left it as ```implementation``` then the bom would be imported and available in compile and runtime scopes as well as test, so you could import into these as well e.g. ```compile("org.testcontainers:testcontainers")```

Reply:
cool.  ➕